packages/orbit-components/src/Tile/components/TileHeader/index.tsx

```diff
@@ -164,27 +74,46 @@ const TileHeader = ({
     tabIndex={tabIndex ? Number(tabIndex) : undefined}
   >
     <Stack align="center" justify="between" shrink spacing="none">
-      {icon && <StyledTileIcon>{icon}</StyledTileIcon>}
+      {icon && (
+        <div className="text-icon-primary-foreground me-xs shrink-0 items-center self-start">
+          {icon}
+        </div>
+      )}
       {header ||
         ((title || description) && (
           <Stack spacing="none" direction="column" shrink>
             {title && (
-              <StyledTileTitle>
+              <div className="flex w-full items-center">
                 <Heading type="title4" as="h3">
                   {title}
                 </Heading>
-              </StyledTileTitle>
+              </div>
             )}
             {description && (
-              <StyledTileDescription hasTitle={!!title}>{description}</StyledTileDescription>
+              <div
+                className={clsx(
+                  "font-base text-normal text-primary-foreground w-full font-normal leading-normal",
+                  title != null && "mt-xxs",
+                )}
+              >
+                {description}
+              </div>
             )}
           </Stack>
         ))}
       {!noHeaderIcon && (
-        <StyledIconRight external={external} expandable={expandable} expanded={expanded} />
+        <IconRight
+          className={clsx(
+            "orbit-tile-header-icon-right text-icon-secondary-foreground ms-sm duration-fast transition-colors ease-in-out",
```

Comment:
This is missing the `transition-property: transform` that applies a transition when the arrow rotates, so I think we need it. I'd just got with `transition-all` since nothing else changes

Reply:
added 👍